bin/kibana

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-exec "${NODE}" $NODE_OPTIONS --no-warnings "${DIR}/src/cli" ${@}
+NODE_ENV=production exec "${NODE}" $NODE_OPTIONS --no-warnings "${DIR}/src/cli" ${@}
```

Comment:
The only thing I can think of here is that we might have server tools that look at `NODE_ENV` and behave "more production-y". I don't immediately see anything in our packages that fit though, only UI related stuff (and those already rely on _our_ `env` through webpack, not `NODE_ENV`).

However, here's at least an example from express: https://github.com/expressjs/express/blob/c087a45b9cc3eb69c777e260ee880758b6e03a40/lib/application.js#L118

However, I think we can merge this as-is, and if we get into any problems re-think the strategy.